set.mm

```diff
@@ -443113,9 +443240,12 @@ the theorem forms ("i", "d" or "g") should be the first of multiple
        <td> ~ nfrmod , ~ nrexrmo </td></tr>
        <tr><td>rn</td><td>range</td><td> ~ df-rn </td><td> ` ran A ` </td>
        <td>Yes</td><td> ~ elrng , ~ rncnvcnv </td></tr>
-       <tr><td>rng</td><td>(unital) ring</td><td> ~ df-ring </td>
+       <tr><td>ring</td><td>(unital) ring</td><td> ~ df-ring </td>
        <td> ` Ring ` </td><td>Yes</td>
        <td> ~ ringidval , ~ isring , ~ ringgrp </td></tr>
+       <tr><td>rng</td><td>(non-unital) ring</td><td> ~ df-rng </td>
```

Comment:
```suggestion
       <tr><td>rng</td><td>non-unital ring</td><td> ~ df-rng </td>
```
When writing for instance "(unital) ring" in a book or article, it implies that in the rest of the article, when writing simply "ring", it will mean "unital ring". Therefore, one cannot write both "(unital) ring" and "(non-unital) ring". Since we indeed want that "ring" alone means "unital ring", we choose to write explicitly "non-unital ring" for those.

Reply:
> When writing for instance "(unital) ring" in a book or article, it implies that in the rest of the article, when writing simply "ring", it will mean "unital ring". Therefore, one cannot write both "(unital) ring" and "(non-unital) ring". Since we indeed want that "ring" alone means "unital ring", we choose to write explicitly "non-unital ring" for those.

Although I agree in principle and support to use "non-unital ring" explicitly if it is not known or not important/necessary that there is a ring unity, I think to use "ring" instead of "non-unital ring" is not wrong, because "non-unital rings" is the more general concept (it includes the unital rings).  Nevertheless, I will remove the brackets as proposed (and look for additional ocurences).